### DISCUSSION TITLE
Only accept the w3mConnectors for mobile wallets?

### DISCUSSION THREAD

**Discussion by danj-O** [WalletConnect/walletconnect-monorepo#2505] [Developer Support]
We are using web3modal in webview in a unity client.  The issue is users keep trying to click the link in modal to download MM browser extension instead of scanning the QR with their mobile.  I'd rather remove those buttons and only give them the QR. Is this possible?

**Comment by finessevanes** [comment]
Are the buttons enclosed within the red outline the ones you're intending to remove?

<img width="376" alt="Screenshot 2023-06-02 at 1 28 56 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/b3f86ea5-bbb1-487c-bc7a-707a4b78b0d7">

**Reply by danj-O** [reply]
Hey there, Yes, I would like to remove all of your outlined options and only allow the QR scan. Reason being is we are using walletconnect in a web-view inside a unity client.  Users get confused and start trying to dl browser extensions in the web-view.

**Reply by danj-O** [reply]
I see there's a unity env label to this, but that's not important.  Web-view is chromium.

A little more info is our app that runs in chromium web-view is a nextjsApp using web3modal, latest updates

**Reply by danj-O** [reply]
Bump

**Reply by glitch-txs** [reply]
hey! This may work for you!
https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/options#explorerrecommendedwalletids-optional

**Reply by danj-O** [reply]
thanks i'll try this out!
